bases/br_sp_alesp/assessores/table_config.yaml

```diff
@@ -0,0 +1,116 @@
+treated_by:
+    name: Diego Oliveira # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_sp_alesp/code # REQUIRED
+    website: https://www.al.sp.gov.br/servidor/lista/?todos=true
+    email: ds.oliveira404@gmail.com
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Diego Oliveira # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_sp_alesp/code # REQUIRED
+    website: github.com/d116626
+    email: ds.oliveira404@gmail.com
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: |
+    Padronização do nome das colunas, remoção de textos e logos, unificação das bases
+
+# Com qual frequência a base é atualizada?
+# Opções: hora | dia | semana | mês | 1 ano | 2 anos | 5 anos | 10 anos | único | recorrente
+data_update_frequency: <frequência> # REQUIRED
+
+# Nível da observação: quais colunas identificam uma linha unicamente?
+# Escolha todas as opções necessárias.
+# Regras: minúsculo, sem acento, singular.
+# Opções (exemplos): pais, estado, municipio, cidade, hora, dia, semana, mes, ano, etc.
+observation_level:
+    - matricula
+    - servidor
+
+# Qual é a cobertura espacial da tabela?
+# Regras:
+#   - minúsculo, sem acento, singular
+#   - descer até o menor nível administrativo cuja cobertura abaixo seja 'todos'
+# Exemplo 1: tabela que cubra todos os municípios nos estados de SP e GO
+#   - brasil
+#   - SP, GO
+# Exemplo 2: tabela que cubra países inteiros na América Latina
+#   - brasil, argentina, peru, equador
+coverage_geo:
+    - brasil
+    - SP
+
+# Qual é a cobertura temporal (em anos) da tabela?
+# Opções: pre-1999, 2000, 2001, ..., 2019, 2020
+coverage_time:
+    - 2020
+
+# Liste as colunas da tabela que representam partições.
+# Não esqueça de deletar essas colunas nas tabelas .csv na hora de subir para o BigQuery.
+# Isso poupará muito tempo e dinheiro às pessoas utilizando essa tabela.
+# Se não houver partições, não modifique abaixo.
+partitions: # REQUIRED
+
+# Quais são as colunas? Certifique-se de escrever uma boa descrição, as pessoas vão gostar
+# para saber sobre o que é a coluna.
+# Adicionar todas as colunas manualmente pode ser bastante cansativo, por isso, quando
+# inicializando este arquivo de configuração, você pode apontar a função para uma amostra de dados que
+# preencherá automaticamente as colunas.
+# Algumas colunas existirão apenas na tabela final, você as construirá em `publish.sql`.
+# Para esses, defina is_in_staging como False.
+# Além disso, você deve adicionar as colunas de partição aqui e definir is_partition como True.
+columns: # REQUIRED
+    - name: matricula
+      description: Número de matrícula do deputado
+      is_in_staging: True # Bool [True, False], whether the column is in the staging table
+      is_partition: False # Bool [True, False], whether the column is a partition.
+
+    - name: nome_parlamentar
```

Comment:
Seria o caso de se chamar só "nome" se for o nome completo da pessoa?

Reply:
então, não é o nome completo, é o nome que ele usa como parlamentar, acabei n mexendo no nome original dessa coluna.